### PR TITLE
Add project description and use case diagram

### PR DESCRIPTION
**Pull request by d9nchik**
What've done in this PR:
- add Use case diagram
- add goals of the project
- add main ideas
- add LICENSE
- add code of conduct
- add ER diagram

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
+Main points:
+
+### User
+
+- can register in system(by pressing button start)
+- can change companion to new one
+- can stop communication
```
Comment (nosovk) [inline README.md:15]: how you will separate communications in chat from commands?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
```
Comment (nosovk) [inline README.md:8]: can we use https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/ for use case diagram?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- block companion to not interact with him anymore
+- get companion "fake" identity (name, age, hobbies)
+- watch own identity
+- edit own identity
+- set random identity (default action on start of the bot)
+
+### Moderator
+
+- can ban user (for inappropriate behaviour(Look more in "inapropriate behavior"))
+- watch logs by user
```
Comment (nosovk) [inline README.md:26]: what data do we store?
do we index that store?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Main points:
+
+### User
+
+- can register in system(by pressing button start)
+- can change companion to new one
+- can stop communication
+- report companion abuse
+- block companion to not interact with him anymore
+- get companion "fake" identity (name, age, hobbies)
```
Comment (nosovk) [inline README.md:18]: how it will be implemented? Some dialog onboarding process?

File: README.md (hunk 2 above)
Comment (d9nchik) [reply README.md:8]: No, we can not:
1. Mermaid [doesn't support](https://mermaid-js.github.io/mermaid/#/sequenceDiagram) use case diagram at all
2. LucidChart provides more enhanced functionality for creating diagrams with WYSIWYG
3. I've already created a diagram

File: README.md (hunk 1 above)
Comment (d9nchik) [reply README.md:15]: the commands will start with `/` sign, the user commonly don't use `/` in their conversation

File: README.md (hunk 4 above)
Comment (d9nchik) [reply README.md:18]: need more clarification

File: README.md (hunk 3 above)
Comment (d9nchik) [reply README.md:26]: I will create ER diagram

File: README.md (hunk 4 above)
Comment (d9nchik) [reply README.md:18]: add sentimental analysis

File: README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
+Main points:
```
Comment (nosovk) [inline README.md:9]: - add Sentiment analysis
- automate moderation
- explain how admin will survive in logs
- add usecase scenarios to use case diagram
- read about data models of chat storage and improve ED diagram

File: README.md (hunk 4 above)
Comment (nosovk) [reply README.md:18]: for example chat with new user starts we can make some short intro with that fake info

File: README.md (hunk 3 above)
Comment (nosovk) [reply README.md:26]: ok

File: README.md (hunk 2 above)
Comment (nosovk) [reply README.md:8]: ok

File: README.md (hunk 1 above)
Comment (nosovk) [reply README.md:15]: ok

File: README.md [hunk 6]
```diff
@@ -60,3 +60,32 @@ We collect messages to provide platform without abuse, advertisement.
...
+- `/ban {user_id}` - give a permanent ban for person. Banned person will be kicked off any present conversation and will get `You're banned` for any command
+- `/user_logs {user_id}` - get logs for user as a txt file
+
+## Admin automation
+
+Before sending messages to admin, they will be send to sentiment analyzer engine. If your text will be evaluated as abuse, you will get ban without calling admin.
+
+### Sentiment analyzer:
+
+We will use Node.JS [sentiment library](https://github.com/thisandagain/sentiment). Scores lower than -3 will earn ban
```
Comment (nosovk) [inline README.md:91]: what you will do with non English chats?

File: README.md [hunk 7]
```diff
@@ -60,3 +60,32 @@ We collect messages to provide platform without abuse, advertisement.
 - information that break copyright law (or any of it analogue) ©
 - advertisement
 - and staff that can violate law
+
+## Scenario of usage
```
Comment (nosovk) [inline README.md:64]: I can't return to previous chat?

File: README.md [hunk 8]
```diff
@@ -60,3 +60,32 @@ We collect messages to provide platform without abuse, advertisement.
...
+
+- `/start` - user get a random identity and get fake identity, status of searching is false
+- `/find_companion` - user status of searching set to true. If match will be found user will get message with companion identity
```
Comment (nosovk) [inline README.md:67]: Do I need to confirm incoming chat inquiry?

File: README.md (hunk 6 above)
Comment (d9nchik) [reply README.md:91]: There is a special part of this library for [adding new languages](https://github.com/thisandagain/sentiment#adding-new-languages)

File: README.md (hunk 7 above)
Comment (d9nchik) [reply README.md:64]: No, you can not. The idea is anonymity and random chat.

File: README.md (hunk 8 above)
Comment (d9nchik) [reply README.md:67]: After we found the chat, the conversation starts automatically, but the user can stop chatting at any moment of time by typing the command `/stop`

File: README.md (hunk 6 above)
Comment (d9nchik) [reply README.md:91]: Also, I found a Russian article [about sentimental analysis and dictionaries comparison](https://www.researchgate.net/publication/349344669_Sovremennye_slovari_ocenocnoj_leksiki_dla_analiza_mnenij_na_russkom_i_anglijskom_azykah_analiticeskij_obzorMODERN_SENTIMENT_LEXICONS_FOR_OPINION_MINING_IN_ENGLISH_AND_RUSSIAN_analytical_survey).
Also, in this article exist dictionaries for Chinese sentimental analysis or Ukrainian.
I think for Russian words I will use [linis-crowd](https://linis-crowd.org/)

**Pull request merged**